src/main/java/frc/robot/Robot.java

```diff
@@ -34,22 +36,25 @@ public class Robot extends LoggedRobot {
     @Override
     public void robotInit() {
         robotContainer = RobotContainer.getInstance();
-        autonomousCommand = robotContainer.getAutonomousCommand();
 
-        Logger.getInstance().recordMetadata("ProjectName", "Wcp-Swerve-2023"); // Set a metadata value
+        Logger.getInstance().recordMetadata("ProjectName", "Robot-template");
 
         if (isReal()) {
-            Logger.getInstance().addDataReceiver(new WPILOGWriter("/media/sda1/")); // Log to a USB stick
-            Logger.getInstance().addDataReceiver(new NT4Publisher()); // Publish data to NetworkTables
-            new PowerDistribution(1, PowerDistribution.ModuleType.kRev); // Enables power distribution logging
+            Logger.getInstance().addDataReceiver(new NT4Publisher());
+            Logger.getInstance().addDataReceiver(new WPILOGWriter("home/lvuser"));
+
+            try (PowerDistribution pwr = new PowerDistribution(1, PowerDistribution.ModuleType.kRev)) {
```

Comment:
Use `LoggedPowerDistribution ` instead. `LoggedDriverStation` and `LoggedSystemStats` exist too.